packages/components/src/components/Card/CardSection.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { forwardRef } from "react";
+import cx from "classnames";
+
+type CardSection = {
+  /** Background color */
+  bg?: "white" | "secondary" | "primary";
+  /** Url becomes value of section's background-image CSS property */
+  bgUrl?: string;
+  /** CSS classes */
+  className?: string;
+  /** Background image is removed for sizes bigger than 768px */
+  hasBgOnMobile?: boolean;
+  /** Condensed card */
+  isCondensed?: boolean;
+  /** Content will fill available vertical space */
+  isFilling?: boolean;
+  /** Type of section */
+  type?: "content" | "image" | "video";

```

Comment:
Tu tie type pre video ostali. Toto by som asi zjednodusil co najviac - ako ti to bude davat zmysel.